### PR TITLE
refactor(pocount): Use csv module for writing csv

### PR DESCRIPTION
**Pull request by last-partizan**
This refactors csv rendering from hand-written style to using csv module.

Output is changed slightly, but this is expected.

### CODE REVIEW

File: translate/tools/pocount.py [hunk 1]
```diff
@@ -240,35 +242,38 @@ def footer(self):
...
+            "fuzzysourcewords": "Fuzzy Source Words",
+            "untranslated": "Untranslated Messages",
+            "untranslatedsourcewords": "Untranslated Source Words",
+            "total": "Total Message",
+            "totalsourcewords": "Total Source Words",
+            "review": "Review Messages",
+            "reviewsourcewords": "Review Source Words",
+        }
+        self._file = io.StringIO()
+        self._writer = csv.DictWriter(self._file, self._fields.values())
```
Comment (nijel) [inline translate/tools/pocount.py:263]: Wouldn't be better to use `sys.stdout` instead of buffering content in `StringIO`?

Comment (last-partizan) [reply translate/tools/pocount.py]: Nice idea, updated code.

**Comment by nijel** [comment]
Merged, thanks for your contribution!

**Comment by last-partizan** [comment]
As the next step i would like to rewrite old shell-based tests for pocount to pytest/snapshot-based tests, but this would require dropping python 3.7.

Do you plan to drop it anytime soon?

**Comment by nijel** [comment]
I don't mind dropping support for Python 3.7 (Weblate will need Python 3.8 from the next release), but please create a separate pull request for that so that other maintainers can express as well.

**Pull request merged**